index.php

```diff
@@ -15,11 +15,25 @@
 <body>
     <main>
         <h1>BEHOLD!! Some deities and stuff:</h1>
-        <div>
+        <div class="displayDeities">
             <?php foreach ($deities as $deity) {
                 echo displayDeity($deity); 
             } ?>
         </div>
+        <div class="addNewDeityForm">
+            <h3>Can't see your favourite deity? No problem! Add them with this form:</h3>
+            <form method="post" action="src/receiveAddNewDeityForm.php">
+                <label>Name:</label><input type="text" name="name">
+                <label>Mythology:</label><input type="text" name="mythology">
+                <label>Association:</label><input type="text" name="association">
+                <label>Object:</label><input type="text" name="objects" placeholder="start with a/an/the">
```

Comment:
1. the labels need the `for` attribute and the input needs the matching `id` attribute
2. it would be more readable to put the input on a new line